test/cog/command/service/data_store_test.exs

```diff
@@ -1,34 +1,30 @@
 defmodule Cog.Command.Service.DataStoreTest do
-  use ExUnit.Case, async: true
+  use ExUnit.Case, async: false
```

Comment:
I didn't want our process-swapping to get messed up (e.g. one test process replace another test process' `DataStore`, etc.)